universal-login-relayer/src/integration/sql/migrations/20200310000001_alter_deployments.js

```diff
@@ -0,0 +1,17 @@
+/*
+  THIS IS A MIGRATION
+  This file SHOULD NOT be RENAMED or CHANGED in ANY WAY!
+  If you need to change the database schema create a new migration.
+*/
+
+exports.up = async (knex) => {
+  await knex.schema.table('deployments', (table) => {
+    table.string('contractAddress', 42).notNullable().defaultTo('0x000000000000000000000000000000');
```

Comment:
@JustynaBroniszewska  What do you think should be a default value?

Reply:
`0x0000000000000000000000000000000000000000`